prices/bsc/coinpaprika.yaml

```diff
@@ -554,3 +554,18 @@
   id: loot-lootex-token
   name: LOOT Token
   symbol: LOOT
+  - address: '0x1FFD0b47127fdd4097E54521C9E2c7f0D66AafC5'
+  decimals: 18
+  id: txl-tixl
```

Comment:
this id appears to only return the ethereum token address, not bsc:
https://api.coinpaprika.com/v1/coins/txl-tixl

are you able to add the bsc address to coinpaprika?
https://coinpaprika.com/add/